test/Conversion/TorchToTosa/basic.mlir

```diff
@@ -1134,3 +1134,47 @@ func.func @torch.aten.remainder.Scalar(%arg0: !torch.vtensor<[2, 4],f32>) -> !to
   %0 = torch.aten.remainder.Scalar %arg0, %int2 : !torch.vtensor<[2, 4],f32>, !torch.int -> !torch.vtensor<[2, 4],f32>
   return %0 : !torch.vtensor<[2, 4],f32>
 }
+
+// -----
+// CHECK-LABEL:   func.func @torch.aten.logical_not(
```

Comment:
when we were adding new legalizations from Torch, we usually didn't add new LIT tests, because functionality tests was mainly done with the python test. So I guess we wouldn't need those LIT tests, but maybe @mgehre-amd knows better.

Reply:
Dropped